src/inngest/functions/create-lesson.ts

```diff
@@ -164,6 +165,18 @@ export let createLesson = inngest.createFunction(
       })
     })
 
+    const {deepgram} = await step.run(
+      'Order Transcript [Deepgram]',
+      async () => {
+        return await orderDeepgramTranscript({
+          moduleSlug: event.data.body._id,
+          mediaUrl: event.data.body.videoResource.videoFile.url,
+          videoResourceId: event.data.body.videoResource._id,
+          eggheadLessonId: lessonObject.id,
+        })
```

Comment:
This will kick off the `transcript-ready` event with necessary data. Added `eggheadLessonId` as a optional prop